tables3_api/tests/test_methods.py

```diff
@@ -0,0 +1,20 @@
+import tables
+from .. import __init__ as tables3_api
```

Comment:
I noticed this first time around...  I am not sure of a better way without playing around, but it's just in the test so no problem.